src/main/java/com/marklogic/kafka/connect/sink/DefaultSinkRecordConverter.java

```diff
@@ -73,7 +105,29 @@ protected AbstractWriteHandle toContent(SinkRecord record) {
 			throw new NullPointerException("'record' must not be null, and must have a value.");
 		}
 		Object value = record.value();
+		Schema schema = record.valueSchema();
+		/* Determine the converter */
+		if (schema != null && value instanceof Struct) {
+			/* Avro, ProtoBuf or JSON with schema, ignore schema, handle only the value */
+			converter = "AVRO_OR_JSON_WITH_SCHEMA";
+			logger.info("Avro or JsonWithSchema document received. Converting to byte[].");
```

Comment:
This seems like debug or more likely trace-level logging. I would either remove it, or if you keep it, provide some context - i.e. some details about the SinkRecord being processed.